charts/argo-cd/Chart.yaml

```diff
@@ -28,3 +28,5 @@ annotations:
   artifacthub.io/changes: |
     - kind: added
       description: Allow configuring Dex's init image resources separately
```

Comment:
Drop the old change. Only your change needs to stay here.

Reply:
Fixed